icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java

```diff
@@ -117,185 +116,251 @@
          */
-        DEKA(1, "deka"),
+        DEKA(1, "deka", 10),
 
         /**
          * The absence of an SI prefix.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        ONE(0, ""),
+        ONE(0, "", 10),
 
         /**
          * SI prefix: deci, 10^-1.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        DECI(-1, "deci"),
+        DECI(-1, "deci", 10),
 
         /**
          * SI prefix: centi, 10^-2.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        CENTI(-2, "centi"),
+        CENTI(-2, "centi", 10),
 
         /**
          * SI prefix: milli, 10^-3.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        MILLI(-3, "milli"),
+        MILLI(-3, "milli", 10),
 
         /**
          * SI prefix: micro, 10^-6.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        MICRO(-6, "micro"),
+        MICRO(-6, "micro", 10),
 
         /**
          * SI prefix: nano, 10^-9.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        NANO(-9, "nano"),
+        NANO(-9, "nano", 10),
 
         /**
          * SI prefix: pico, 10^-12.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        PICO(-12, "pico"),
+        PICO(-12, "pico", 10),
 
         /**
          * SI prefix: femto, 10^-15.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        FEMTO(-15, "femto"),
+        FEMTO(-15, "femto", 10),
 
         /**
          * SI prefix: atto, 10^-18.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        ATTO(-18, "atto"),
+        ATTO(-18, "atto", 10),
 
         /**
          * SI prefix: zepto, 10^-21.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        ZEPTO(-21, "zepto"),
+        ZEPTO(-21, "zepto", 10),
 
         /**
          * SI prefix: yocto, 10^-24.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        YOCTO(-24, "yocto");
+        YOCTO(-24, "yocto", 10),
+
+        /**
+         * IEC binary prefix: kibi, 1024^1.
+         *
+         * @draft ICU 69
+         * @provisional This API might change or be removed in a future release.
+         */
+        KIBI(1, "kibi", 1024),
```

Comment:
I prefer instead of using the number. use a constant. 
Using constant is more illustrative and error-prone. it is easy to miss type a number.

for example: `MeasurePrefix.BINARY_BASE`